docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst

```diff
@@ -71,28 +47,26 @@ In particular, we will examine:
 - Impairments: heart failure 
 - Sequelae: numerous, including "severe heart failure due to IHD" and "AMI first 2 days" among others 
 
-Causes
-++++++
-
-The cause hierarchy diagrams for IHD and HHD are included here as a reference. 
+To help with understanding some of the below information, a diagram of the heart failure impairment is provided: 
 
+.. image:: health_state_impairment_sequela.svg
 
-**IHD**:
 
-.. image:: cause_hierarchy_ihd.svg
+Causes
+++++++
 
-**HHD**: 
+The cause hierarchy diagram for IHD and HHD is included here as a reference. 
 
-.. image:: cause_hierarchy_hhd.svg
+.. image:: cause_hierarchy_ihd_hhd.svg
 
 As you can see, the cause hierarchy slowly breaks down cause groups into more specific 
 diseases, moving from all non-communicable diseases to specific types of cardiovascular 
-disease. At each level, the cause hierarchy is mutually exclusive and 
+disease. At each level (labeled), the cause hierarchy is mutually exclusive and 
 collectively exhaustive - meaning all disease and death can be placed into a group at 
-each level of the diagram. 
+each level of the diagram. A cause is anything that causes death or disability. 
 
 Both IHD and HHD are within the more general cardiovascular disease cause (c_491). IHD and HHD 
-are causes within GBD. 
+are "causes" within GBD. 
 
 Sequelae
```

Comment:
I think it would be useful to add here what we discussed about the difference between causes and sequelae: that sequelae are almost like the last level of the cause hierarchy, except that they are not modeled as complexly. Specifically, that the data used to model sequelae are generally not differentiated by location, year, age or sex (is this correct)?

Reply:
I added some of this but am not 100% sure that there aren't cases where sequela modeling is different by age/sex/etc. so am hedging a bit in language.